neural_mfg.py

```diff
@@ -10,7 +10,7 @@
 import os
 import json
 import matplotlib
-matplotlib.use('Agg')
+#matplotlib.use('Agg')
 import matplotlib.pyplot as plt
 import datetime
 
```

Comment:
We found that for $\gamma > 0$ $\lambda$ should be equal to 0 . Maybe an if can be added .

Reply:
I think it is the inverse (here you write when $\gamma > 0$ $\lambda = -gm_0$)